src/metabase/pulse/render/body.clj

```diff
@@ -649,18 +648,16 @@
             card-name          (or (series-setting viz-settings group-key :name)
                                    (series-setting viz-settings group-key :title)
                                    group-key)
-            card-color         (or (series-setting viz-settings group-key :color)
-                                   (nth colors idx))
             card-type          (or (series-setting viz-settings group-key :display)
                                    chart-type
                                    (nth default-combo-chart-types idx))
             y-axis-pos         (or (series-setting viz-settings group-key :axis)
                                    (nth (default-y-pos viz-settings) idx))]
         {:name          card-name
-         :color         card-color
          :type          card-type
          :data          selected-row-group
-         :yAxisPosition y-axis-pos}))))
+         :yAxisPosition y-axis-pos
+         :column        (second x-cols)}))))
```

Comment:
For charts with breakout values, FE will set the series key (`String(single.card.name)`) for calculating colors using the each breakout value with this logic
- https://github.com/metabase/metabase/blob/d9a5708306f464e5d512c5d2b7cd18677a1f7667/frontend/src/metabase/visualizations/lib/settings/series.js#L12
- https://github.com/metabase/metabase/blob/d9a5708306f464e5d512c5d2b7cd18677a1f7667/frontend/src/metabase/visualizations/components/LineAreaBarChart.jsx#L482

See the example chart below which explains why `:column` is needed here.

![image](https://user-images.githubusercontent.com/1937582/199700608-303a3095-e676-488f-870e-c206f6527ffd.png)

FE will receive the value like `2016-01-01T00:00:00Z` and pass that along with `column` to `formatValue` and produce the value `2016`. Since this column's type is date, the value is formatted by moment.js.

This is why we need to pass `column` from BE so FE static viz could produce the same series key FE viz does results in the same colors, since these series key are then going to go through some logic which calculate all series colors based on the overall series keys.
https://github.com/metabase/metabase/blob/d9a5708306f464e5d512c5d2b7cd18677a1f7667/frontend/src/metabase/visualizations/lib/settings/series.js#L193